### ISSUE TITLE
Deploy examples to another robot

### ISSUE THREAD

**Issue by JiangZhebin** [hnuzhy/YOTO#3]
Hello,
I'm trying to deploy the pretrained model to my bimanual robot with a zed2i camera. But it appears that the camera frame in your pretrained model is different from the zed_left_camera_optical_frame and the ee_frames seems not clearly defined.

Could you please clarify how is the camera frame and end-effector frames defined in your examples? 

I am also curious about how to create my own data. According to the paper, it seems that I can generate training dataset using one simple demonstration video with my own camera. Are you going to release the data generation and preprocess code? 

Best regards,
Zhebin

**Comment by hnuzhy** [comment]
Thank you for your interest. For the third-view camera pose, you can refer [here - cfg_dict_init](https://github.com/hnuzhy/YOTO/blob/main/AugDemos/aug_dataset_v1.py#L15). As for the initial robot pose of the left/right arm, you can find them in the released [dataset - taskname_preprocessed.json](https://huggingface.co/HoyerChou/YOTO). The specific format of these json files and the scripts to generate demonstrations can be found in [aug_dataset_v1.py](https://github.com/hnuzhy/YOTO/blob/main/AugDemos/aug_dataset_v1.py#L143) and [aug_dataset_v2.py](https://github.com/hnuzhy/YOTO/blob/main/AugDemos/aug_dataset_v2.py#L80).

**Comment by JiangZhebin** [comment]
Thank you for your reply.
I double checked the demo video and found that all the coordinate systems used in the video are left-handed, which is different from the ROS2 right-handed standard. Just want to make sure if all the coordinate systems used in your project are left-handed?

**Comment by hnuzhy** [comment]
When extracting the 3D pose, I first calculated the `z`-axis, which is the direction obtained by the cross-product of the vector from the wrist point to the two fingertips (index finger and ring finger) (blue line), and then determined the `y`-axis (green line) and `x`-axis (red line) (refer to Alg.1 in the paper). Due to the chirality of the left and right hands, the `z`-axis of the left hand is perpendicular to the palm facing inward, and the `z`-axis of the right hand is perpendicular to the palm facing outward. However, it is feasible to convert these keyposes into 6DoF poses in the robot coordinates according to the hand-eye calibration matrix (extra attention is required).
